lib/monthly.rb

```diff
@@ -0,0 +1,118 @@
+# frozen_string_literal: true
+
+module Declarations
+  module Prefills
+    module Prefillables
+      class MonthlyVatReturnService < ApplicationService
+        SENTRY_OWNER = Constants::SentryOwners::DECLARATIONS
+
+        private
+
+        IMPORTATION_VAT_RATES = [0.2, 0.1, 0.085, 0.055, 0.021, 0.0105].freeze
+
+        def initialize(declaration:, prefill:)
+          @declaration = declaration
+          @declaration_without_prefills = @declaration.dup
+          @prefill = prefill
+        end
+
+        def call
+          prefill_cells
+          prefill_cells_tax
+          recalculate_cell16
+          recalculate_cell20
+          recalculate_cell23
+          recalculate_cell24
+
+          @declaration
+        end
+
+        def prefill_cells
```

Comment:
The method `prefill_cells` seems to be formatting data rather than encapsulating business logic. Ensure that this action is complex enough to warrant a service.